package.json

```diff
@@ -102,8 +91,8 @@
         "eslint-plugin-prettier": "^3.1.2",
         "eslint-plugin-react": "^7.17.0",
         "eslint-plugin-react-hooks": "^1.7.0",
-        "flow-bin": "^0.79.0",
-        "flow-typed": "^2.5.1",
+        "flow-bin": "^0.95.1",
```

Comment:
I upgraded to the same version of flow as box-ui-elements in order to support React Hooks.